### PR TITLE
LGAT subtyping update 

### PR DESCRIPTION
**Pull request by runjin326**
### Purpose
LGAT subtyping module added to the method section

### Issue

https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/136

#### Which areas should receive a particularly close look?
Please check to see whether the recoding part is clear enough.

#### Is there anything that you want to discuss further?
No.

#### Is the pull request ready for review?
Yes.

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/03.methods.md
Comment (jaclyn-taroni) [inline content/03.methods.md:401]: I would expect the abbreviation here to be LGG based on the terminology used. What's the correct terminology here? cc: @jharenza

Comment (jaclyn-taroni) [reply content/03.methods.md:401]: Where might appropriate citations from https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/790 ([Ryall et al. [1]](https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-00902-z), [Ryall et al [2]](https://www.sciencedirect.com/science/article/pii/S1535610820301513)) go?

**Review by jaclyn-taroni — COMMENTED** [review]
The description seems to be in line with the README. I have questions about terminology and citations, but I don't think I'm the best person to answer them so I will remove my request for review and add @jharenza

File: content/03.methods.md
Comment (jharenza) [reply content/03.methods.md:401]: These abbreviations are always hard, and I initially used LGAT/HGAT because astrocytomas and gliomas all arise from astrocytes. Though, most of the field uses pLGG and pHGG (pediatric LGG or HGG). So for this, I would suggest using 
`Any low-grade astrocytic tumor (LGAT) that contained...` 

I think we can defer to clinicians (I will ask) whether we should use pLGG or LGAT.

File: content/03.methods.md
Comment (jharenza) [inline content/03.methods.md:417]:
```suggestion
If pathology diagnosis was `Subependymal Giant Cell Astrocytoma (SEGA)`, the `LGG` portion of molecular subtype was recoded to `SEGA`.

Lastly, for all subtyped samples, if the tumors were glialneuronal in origin, based on `pathology_free_text_diagnosis` entries of `desmoplastic infantile`,`desmoplastic infantile ganglioglioma`, `desmoplastic infantile astrocytoma` or `glioneuronal`, each was recoded as follows:
```

SEGA is not a GNT, so I moved this one up

File: content/03.methods.md
Comment (jharenza) [inline content/03.methods.md:421]:
```suggestion
If pathology diagnosis is `Low-grade glioma/astrocytoma (WHO grade I/II)` or `Ganglioglioma`, the `LGG` portion of the molecular subtype was recoded to `GNT`.
```

**Review by jharenza — CHANGES_REQUESTED** [review]
thanks @runjin326 for updating this! I just have a few change requests

File: content/03.methods.md
Comment (jharenza) [reply content/03.methods.md:401]:
```suggestion
Low-grade astrocytic tumors (LGAT) or glialneuronal tumors (GNT) were subtyped based on SNV, fusion and CNV status based on @doi:10.1016/j.ccell.2020.03.011, and as described below.
```

I took out some details

File: content/03.methods.md
Comment (runjin326) [reply content/03.methods.md:402]:
```suggestion
1. If a sample contained a _NF1_ somatic mutation, either nonsense or missense, it was subtyped as `LGG, NF1-somatic`.
```

**Comment by runjin326** [comment]
@jharenza, thanks for reviewing this! I committed your change suggestions!

**Review by jharenza — APPROVED** [review]
sorry i missed this re-review, approving now

**Pull request merged**
